sdk/servicebus/Azure.Messaging.ServiceBus/tests/Processor/ProcessorLiveTests.cs

```diff
@@ -1064,6 +1064,69 @@ async Task ProcessMessage(ProcessMessageEventArgs args)
             }
         }
 
+        [Test]
+        public async Task CanUpdatePrefetchCount()
```

Comment:
I really struggled to come up with a good test

Reply:
Outside of doing some some sort of perf comparison, I think this is the best we can do for a live test.